src/docs/develop/cli.md

```diff
@@ -7,84 +7,122 @@ alt="GIF of the CLI in Action"
 layout="intrinsic"
 width={800} height={468} quality={100} />
 
-The Railway command line interface (CLI) connects your code to your Railway project from the command line.
+The Railway Command Line Interface (CLI) lets you interact with your
+Railway project from the command line, allowing you to:
 
-The Railway CLI allows you to
+- Create new Railway projects from the Terminal
+- Link a local project directory to an existing Railway project
+- Run services locally using environment variables from your Railway project
+- And more!
 
-- Create new Railway projects from the terminal
-- Link to an existing Railway project
-- Pull down environment variables to run a service locally
-- Open an interactive shell to a database within a project
+## Installation
 
-## Install
+The Railway CLI can be installed via:
 
-Install with [Brew](https://brew.sh), [NPM](https://www.npmjs.com/package/@railway/cli), Shell Script, or [Scoop](https://scoop.sh).
+- [Homebrew](https://brew.sh) for macOS
+- [npm](https://www.npmjs.com/package/@railway/cli) for macOS, Linux, and Windows
+- [Scoop](https://scoop.sh) for Windows as a native `.exe` binary
+- Direct Shell Script
+- Downloading pre-built binaries
+- Building from source
 
-### NPM (Linux, macOS, Windows)
+### Homebrew (macOS)
```

Comment:
> I don't think this is entirely inaccurate. With that said, maybe we should link to the `brew` docs for distros that we don't support.
> 
> https://docs.brew.sh/Homebrew-on-Linux

...interesting! Is brew a popular package manager on Linux? While it's technically true and accurate, I've never seen it recommended/specified as a installation method anywhere. 

> ...Plus maybe caveat by saying `WSL` instead of Windows.

4a073dbcade0fd5e5a82cbfebe68c21fff783095